cms/djangoapps/contentstore/views/component.py

```diff
@@ -131,7 +131,7 @@ def subsection_handler(request, usage_key_string):
             }
         )
     else:
-        return HttpResponseBadRequest("Only supports html requests")
+        return HttpResponseBadRequest(_("Only supports html requests"))
```

Comment:
This string doesn't really make sense out of context. Can you please add a translators comment explaining what it means.